src/utils.js

```diff
@@ -3,23 +3,23 @@ const process = require('process');
 const figlet = require('figlet');
 const chalk = require('chalk');
 const constants = require('./constants');
+const utils = require('../src/utils');
```

Comment:
Mm.. seems like a circular dependency here.. Not sure whether some Node.js versions display warnings on console about this import. Could you please check further about this and check whether there is another way to organize these functions?